pkg/api/types.go

```diff
@@ -2979,6 +2979,9 @@ type AttachedVolume struct {
 
 	// DevicePath represents the device path where the volume should be available
 	DevicePath string
+
+	// Has the process of attaching the volume finished already?
+	IsAttached bool
```

Comment:
About the attribute name: you're right, it looks rather odd -- I should have called it AttachFinished or something. Having parallel array of volumes being attached has been discussed in the original issue. It felt wasteful to me so I've tried this first. Having one array of Volumes with all the data required for crash recoveries etc., has been suggested too. It would be a rather big change though and I would have to keep the old VolumesInUse and VolumesAttached for compatibility reasons anyway. So I guess I will have to add another array to the node status...

Thanks for the review.